### PR TITLE
Add meeting files for TAC meeting 2024-09-19

### PR DESCRIPTION
**Pull request by riaankleinhans**
@renuchauhan221

### CODE REVIEW

File: TAC/Meetings/2024/2024-09-19/CCC-TAC-Minutes-2024-09-19.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+Dan Middleton (DM) opened the call at 7:04 am PT and reviewed the antitrust policy.
+
+## Agenda
+
+DM reviewed the agenda for the meeting.
+
+## Roll Call / Attendance
+
+Quorum was established as no votes were required.
```
Comment (dcmiddle) [inline TAC/Meetings/2024/2024-09-19/CCC-TAC-Minutes-2024-09-19.md:87]:
```suggestion
Quorum was not established as no votes were required.
```

File: TAC/Meetings/2024/2024-09-19/CCC-TAC-Minutes-2024-09-19.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+* Caroline Perez-Vargas from Microsoft  introduced themselves.
+
+## Old Business & Announcements
+
+DM reviewed the old business from the previous meeting.
+
+## Announcements
+
+Sal announced Kubconf Salt Lake City in mid Nov and encouraged members to reach to participate in the Kubcon activities.
```
Comment (dcmiddle) [inline TAC/Meetings/2024/2024-09-19/CCC-TAC-Minutes-2024-09-19.md:99]:
```suggestion
Sal announced Kubconf Salt Lake City in mid Nov and encouraged members to reach out to participate in the Kubcon activities.
```

File: TAC/Meetings/2024/2024-09-19/CCC-TAC-Minutes-2024-09-19.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+## Attestation Terminology
+
+Mike Bursell (MB) introduced Attestation & AVS (Attestation Verification Services)
```
Comment (dcmiddle) [inline TAC/Meetings/2024/2024-09-19/CCC-TAC-Minutes-2024-09-19.md:103]:
```suggestion
```

File: TAC/Meetings/2024/2024-09-19/CCC-TAC-Minutes-2024-09-19.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+## Formal programming techniques Tech Talk
+
+Patrick Eugster- Formal programming techniques for secure data processing
+[LINK TO PRESENTATION](./STYX_CCC_TAC_2024.pdf)
+
+## Repositioning WG
+
+Ab Nacef introduced the meeting to the work being done by the working group.
+Julian Stephen walked through the three main focus items of the work group being the Messaging Guide around Use Cases.Emphasized on Supply Chain Security Use case and requested for any familiar scenarios from the community by mid October and ASAP information is appreciated. Discussion ensued.
```
Comment (dcmiddle) [inline TAC/Meetings/2024/2024-09-19/CCC-TAC-Minutes-2024-09-19.md:118]:
```suggestion
Julian Stephen walked through the three main focus items of the work group being the Messaging Guide around Use Cases. Emphasized Supply Chain Security Use case and requested any familiar scenarios from the community by mid-October and ASAP information is appreciated. Discussion ensued.
```

File: TAC/Meetings/2024/2024-09-19/CCC-TAC-Minutes-2024-09-19.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+
+## Attestation & AVS Questions
+
+DM reviewed the topics raised by Mike Bursell in the previous meeting. DM continued discussing the roles in the TCB and gave a brief overview . He encourage further discussion of the topic in the TAC mailing list.
```
Comment (dcmiddle) [inline TAC/Meetings/2024/2024-09-19/CCC-TAC-Minutes-2024-09-19.md:122]:
```suggestion
DM reviewed the topics raised by Mike Bursell in the previous meeting. DM continued discussing the roles in the TCB and gave a brief overview.
He encourage further discussion of the topic in the TAC mailing list.
```

File: TAC/Meetings/2024/2024-09-19/CCC-TAC-Minutes-2024-09-19.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+## Sticky Policy Tech Talk
+
+Zhiqiang - Collaborative and Private Data Processing with TEE-enforced Sticky Policy
+[LINK TO PRESENTATION](./ConfCompBeyondClouds-CCC.pdf)
+
+## Formal programming techniques Tech Talk
+
+Patrick Eugster- Formal programming techniques for secure data processing
+[LINK TO PRESENTATION](./STYX_CCC_TAC_2024.pdf)
+
```
[the remark this answers is not in the document]
Comment (riaankleinhans) [reply TAC/Meetings/2024/2024-09-19/CCC-TAC-Minutes-2024-09-19.md:111]: Added

**Review by dcmiddle — APPROVED** [review]

**Pull request merged**